backend.py

```diff
@@ -687,21 +722,26 @@ def add_to_past_deck(self, cards):
         for card in cards:
             self.past_deck.add(card)
 
-    def cards_and_game_round_as_dict(self, cards):
+    def cards_and_game_round_as_dict(self, cards, blocked):
```

Comment:
Zde me zmatl nazev argumentu: blocked je moc vagni. Chvili jsem uvazovala, jestli je to bool hodnota (blokovany - true/false). Prosim blocked_cards nebo neco podobneho.